src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/IntentionalRerouteSpec.groovy

```diff
@@ -239,19 +246,20 @@ class IntentionalRerouteSpec extends HealthCheckSpecification {
         def switchPair = switchPairs.all().neighbouring().withBothSwitchesVxLanEnabled().withTraffgensOnBothEnds().random()
         def availablePaths = switchPair.paths.findAll { pathHelper.getInvolvedSwitches(it).find { it.noviflow }}
 
-        def flow = flowHelperV2.randomFlow(switchPair)
-        flow.maximumBandwidth = 0
-        flow.ignoreBandwidth = true
-        flow.encapsulationType = FlowEncapsulationType.VXLAN
-        flowHelperV2.addFlow(flow)
-        def altPaths = availablePaths.findAll { it != pathHelper.convert(northbound.getFlowPath(flow.flowId)) }
+        def flow = flowFactory.getBuilder(switchPair)
+                .withBandwidth(0)
+                .withIgnoreBandwidth(true)
+                .withEncapsulationType(FlowEncapsulationType.VXLAN)
+                .build().create()
+        assert pathHelper.convert(northbound.getFlowPath(flow.flowId)) == flow.retrieveAllEntityPaths().getPathNodes()
```

Comment:
what is the purpose of this check?

Reply:
I think I was debugging something and forgot to remove this line 😄